easybuild/easyconfigs/c/Cython/Cython-0.29.22-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,54 @@
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 The Cyprus Institute
+# Authors::   Andreas Panteli <a.panteli@cyi.ac.cy>,
+#             Thekla Loizou <t.loizou@cyi.ac.cy>,
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# License::   MIT/GPL
+#
+# Updated: Pavel Grochal (INUITS)
+##
+# Thomas Hoffmann, EMBL Heidelberg: Adapted to v0.29.22, GCCcore 10.2.0
+# 2021/03
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '0.29.22'
+
+homepage = 'https://cython.org/'
+description = """
+Cython is an optimising static compiler for both the Python programming
+language and the extended Cython programming language (based on Pyrex).
+"""
+docurls = [
+    'https://cython.org/#documentation',
+    'https://github.com/cython/cython',
+]
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['df6b83c7a6d1d967ea89a2903e4a931377634a297459652e4551734c48195406']
+
+builddependencies = [('binutils', '2.36.1')]
+
+# Can't use multi_dep because EBPYTHONPREFIXES are not loaded in order.
+# This results in not beeing able to choose Cython version in multi_dep.
```

Comment:
I suppose we can remove this comment because since GCCcore/10.3.0 we stopped using `multi_deps` for python 2 and 3 anyway